bundles/specmate-integration-test/resources/DE/Dep_DE.spec

```diff
@@ -0,0 +1,226 @@
+def tagset dependencies DE.DEP.NEGRA {
```

Comment:
Is it necessary to have the rules in the test package? Then we have them redundantly at two places. If we change a rule in the model generation package and don't change it in the test package, the tests will not reflect the current state.